Assets/Scripts/NightOneManager.cs

```diff
@@ -0,0 +1,32 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class NightOneManager : MonoBehaviour
+{
+    [SerializeField] AudioSource[] audioSources;
+    private int index;
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        index = 0;
+        StartCoroutine(StartAmbientMusic());
+    }
+
+    private IEnumerator StartAmbientMusic()
+    {
+        double offset = 0.01;
+        double length = audioSources[0].clip.length;
+        double time = AudioSettings.dspTime - offset;
+        audioSources[index].Play();
+
+        while (true)
```

Comment:
Maybe add a stop condition here? I think this should be fine once we change scenes, but it might be good to be sure it will stop